publication-commons/src/main/java/no/unit/nva/publication/model/business/publicationstate/ResourceEvent.java

```diff
@@ -10,10 +10,11 @@
 import no.unit.nva.publication.model.business.logentry.PublicationLogEntry;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-@JsonSubTypes({@JsonSubTypes.Type(CreatedResourceEvent.class), @JsonSubTypes.Type(PublishedResourceEvent.class),
-    @JsonSubTypes.Type(UnpublishedResourceEvent.class), @JsonSubTypes.Type(DeletedResourceEvent.class),
-    @JsonSubTypes.Type(RepublishedResourceEvent.class), @JsonSubTypes.Type(ImportedResourceEvent.class),
-    @JsonSubTypes.Type(DoiReservedEvent.class), @JsonSubTypes.Type(MergedResourceEvent.class)})
+@JsonSubTypes({@JsonSubTypes.Type(CreatedResourceEvent.class), @JsonSubTypes.Type(UpdatedResourceEvent.class),
+    @JsonSubTypes.Type(PublishedResourceEvent.class), @JsonSubTypes.Type(UnpublishedResourceEvent.class),
+    @JsonSubTypes.Type(DeletedResourceEvent.class), @JsonSubTypes.Type(RepublishedResourceEvent.class),
+    @JsonSubTypes.Type(ImportedResourceEvent.class), @JsonSubTypes.Type(DoiReservedEvent.class),
+    @JsonSubTypes.Type(MergedResourceEvent.class)})
```

Comment:
Wondering if we should sort this list alphabetically and do one-statement-per-line separation. so we can see the actual update when the code changes